DOC-2463: Added new option `advcode_prettify_getcontent` to Enhanced Code Editor. (#3400)
tinymce/tinymce-docs: modules/ROOT/pages/7.3-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Enhanced Code Editor

The {productname} {release-version} release includes an accompanying release of the **Enhanced Code Editor** premium plugin.

**Enhanced Code Editor** includes the following improvements.

==== Added new option `advcode_prettify_getcontent` to Enhanced Code Editor.

> Previously, formatting was only applied in the **Enhanced Code Editor** view, and as a consequence users could only obtain raw, unformatted HTML when calling `editor.getContent` via the editor API.

Reviewer: ```suggestion
Previously, formatting of HTML content was only applied in the **Enhanced Code Editor** view, and as a consequence, it was not possible to obtain formatted HTML content when calling the `editor.getContent` API.
```

> Previously, formatting of HTML content was only applied in the **Enhanced Code Editor** view, and as a consequence, it was not possible to obtain formatted HTML content when calling the `editor.getContent` API.

Reviewer 2: Both functionalities - formatted code in the `Enhanced Code Editor` view and getting formatted code on `editor.getContent` API are introduced in 7.3 so I am not sure about phrasing it like that. Sound like the first one was there before, which is not true.

Author: ```suggestion
Previously, formatting HTML content within the **Enhanced Code Editor** view or retrieving formatted HTML from the `editor.getContent` API as not possible.
```

> {productname} {release-version} introduces the new `advcode_prettify_getcontent` option, which resolves this issue. When enabled (default is false), calling `editor.getContent` now returns formatted HTML. Additionally, it is now possible to retrieve formatted content by calling `editor.getContent({ prettify: true })`, regardless of the option's status.

Author: ```suggestion
To resolve this issue, {productname} {release-version} introduces a new `advcode_prettify_getcontent` option. When enabled (default: false), `editor.getContent` will return formatted HTML. Additionally, invoking `+editor.getContent({ prettify: true })+` will retrieve formatted HTML content regardless of the option's status.
```
